packages/client/src/domain/Kayenta.ts

```diff
@@ -243,7 +243,7 @@ export interface StageMetadata {
   type: string;
   name: string;
   status: string;
-  executionId: string;
+  executionId?: string;
```

Comment:
Does this mean executionId is optional?

Reply:
Yes, `executionId` is only present if the stage is a [`runCanary` stage](https://github.com/spinnaker/kayenta/commit/4fe4ea5c55cc5c21b34e482ce93da5b4302eef29#diff-16744a536ace9d5bafab8023ee6d4096R48). Stages like `setupAndExecuteCanariesStage` and `generateCanaryAnalysisResultStage` won't have an `executionId`.